src/containers/ListingPage/SectionFeaturesMaybe.js

```diff
@@ -18,7 +18,9 @@ const SectionFeaturesMaybe = props => {
       </h2>
       <PropertyGroup
         id="ListingPage.yogaStyles"
-        options={options}
+        options={options.filter(o => {
+          return selectedOptions.find(s => s === o.key);
+        })}
         selectedOptions={selectedOptions}
         twoColumns={true}
```

Comment:
If you create variable out of filter 
(e.g. `const selectedConfigOptions = options.filter(o => selectedOptions.find(s => s === o.key);` 
you could add here something like:
`twoColumns={selectedConfigOptions > 5}`